src/service-providers/decision-trees/decision-trees.types.ts

```diff
@@ -0,0 +1,7 @@
+import { TItemsCount, TPropertyStatus } from '../common'
```

Comment:
move types to decision-trees.interface.ts

Reply:
it's a common types from common.intefaces.ts, only imported from index.ts